### PR TITLE
[MM-29725] When a user joins the server, update user count (and more)

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
Currently in cloud-ga branch, if a user joins the server, the admin is not receiving the banner to upgrade mattermost-cloud until they refresh. This PR adds a check on the initial channel joins to refresh the analytics. It will only refresh analytics if it is a cloud installation, and only for admin users, in order to not tax the backend.

Additionally, this PR includes a consumer of a new websocket event (see https://github.com/mattermost/mattermost-server/pull/15990) that is triggered when a user is activated or deactivated. This means that deactivating enough users to go under the limit will automatically update the banners, and activating users to go over the limit will also update the banners.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-29725

#### Related Pull Requests

- Has server changes https://github.com/mattermost/mattermost-server/pull/15990

#### Screenshots
N/A

### CODE REVIEW

File: actions/websocket_actions.jsx [hunk 1]
```diff
@@ -472,6 +473,10 @@ export function handleEvent(msg) {
     case SocketEvents.SIDEBAR_CATEGORY_ORDER_UPDATED:
         dispatch(handleSidebarCategoryOrderUpdated(msg));
         break;
+    case SocketEvents.USER_ACTIVATION_STATUS_CHANGED:
+        console.log('HI');
```
Comment (devinbinnie) [inline actions/websocket_actions.jsx:477]: Probably didn't mean to commit this? :P

**Review by devinbinnie — APPROVED** [review]

File: actions/websocket_actions.jsx (hunk 1 above)
Comment (nickmisasi) [reply actions/websocket_actions.jsx]: lol woops

**Review by marianunez — APPROVED** [review]

**Comment by nickmisasi** [comment]
@stevemudie this one isn't testable pre-merge due to the dependency on CWS. will need to be tested post merge.

**Comment by amyblais** [comment]
@nickmisasi This and https://github.com/mattermost/mattermost-server/pull/15990 can be cherry-picked to cloud-ga.

**Pull request merged**